spec/frequency_classes_service_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'frequency_classes_service'
+
+describe 'FrequencyClassesService' do
+  let(:frequency_classes_service) { FrequencyClassesService.new }
+  let(:sample_word) { 'haben' }
+
+  describe '#initialize' do
+    it 'should return a FrequencyClassesService instance' do
+      expect(frequency_classes_service).to be_instance_of(
+        FrequencyClassesService
+      )
+    end
+  end
+
+  describe '#calculate' do
+    it 'should calculate the frequency class for haben to match 3' do
```

Comment:
no i mean what happens for other words? does it take multiple words? show a variety of ways to use the method that are valid. if there are ways that are invalid, specify those boundaries as well ("requires an argument that responds to `to_s`" or "requires a string" or "raises ArgumentError unless supplied with a string or array of strings" or whatever is true and helpful to know about this code interface)